fenixedu-ist-teacher-credits/src/main/resources/META-INF/web-fragment.xml

```diff
@@ -1,45 +0,0 @@
-<?xml version="1.0" encoding="UTF-8"?>
-<!--
-
-    Copyright © 2013 Instituto Superior Técnico
-
-    This file is part of FenixEdu IST Teacher Credits.
-
-    FenixEdu IST Teacher Credits is free software: you can redistribute it and/or modify
-    it under the terms of the GNU Lesser General Public License as published by
-    the Free Software Foundation, either version 3 of the License, or
-    (at your option) any later version.
-
-    FenixEdu IST Teacher Credits is distributed in the hope that it will be useful,
-    but WITHOUT ANY WARRANTY; without even the implied warranty of
-    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
-    GNU Lesser General Public License for more details.
-
-    You should have received a copy of the GNU Lesser General Public License
-    along with FenixEdu IST Teacher Credits.  If not, see <http://www.gnu.org/licenses/>.
-
--->
-<web-fragment xmlns="http://java.sun.com/xml/ns/javaee" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
-    xsi:schemaLocation="http://java.sun.com/xml/ns/javaee http://java.sun.com/xml/ns/javaee/web-fragment_3_0.xsd"
-    version="3.0">
-    <name>${artifactId}</name>
-    <description>${name}</description>
-
-    <ordering>
-        <before>
-            <others />
-        </before>
-    </ordering>
-
-	<!-- Filter -->
```

Comment:
The web-fragment file is still needed, only the unnecessary filter should go out